exercises/isbn-verifier/.meta/src/reference/java/IsbnVerifier.java

```diff
@@ -0,0 +1,36 @@
+class IsbnVerifier {
+
+    boolean isValid(String stringToVerify) {
+
+        String isbn;
+        isbn = stringToVerify.replace("-", "");
+        int total = 0;
+
+        if (isbn.length() != 10) {
+            return false;
+        }
+
+        for (int i = 0; i < isbn.length() - 1; i++) {
+            char currentDigit = isbn.charAt(i);
```

Comment:
Maybe `currentChar`? Either that or use `currentDigitVal` and `finalDigit` later to be consistent :)